code/datums/mood_events/generic_positive_events.dm

```diff
@@ -293,3 +293,11 @@
 
 	mood_change *= card_players
 	return ..()
+
+/datum/mood_event/russian_roulette_win
+	description = "I gambled my life and won! I'm lucky to be alive..."
+	mood_change = 2
+	timeout = 5 MINUTES
```

Comment:
idk how i feel about the mood buff being so long tbh, you have  a fairly high chance of not dying on the first shot in russian roulette so this just feels like an easy way to get mood

Reply:
There are plenty of other positive events that give around the same mood buff without the possibility of dying.  It's also offset by a severe negative mood if they lose and get revived.